### PR TITLE
Issue with variant

### PR DESCRIPTION
**Pull request by ChHaeni**
@jnthn this code fails for model variant `Single_extendedOutput`. It runs through if either variant is other than `Single_extendedOutput` or if the `?if` statement is removed/commented. The following error occurs if variant `Single_extendedOutput` is run:
```
# Failed test 'Load module End.nhd with variant Single_extendedOutput from /home/christoph/repos/5_GitHub/agrammon-workbench/agrammon/t/test-data/Models/hr-inclNOxExtendedWithFilters/'                                
# at /home/christoph/repos/5_GitHub/agrammon-workbench/agrammon/t/run-complex-model-with-variants.t line 46
# Cannot look up attributes in a Agrammon::Model::Output type object
```

### CODE REVIEW

**Comment by ChHaeni** [comment]
> @jnthn this code fails for model variant `Single_extendedOutput`. It runs through if either variant is other than `Single_extendedOutput` or if the `?if` statement is removed/commented. The following error occurs if variant `Single_extendedOutput` is run:
> 
> ```
> # Failed test 'Load module End.nhd with variant Single_extendedOutput from /home/christoph/repos/5_GitHub/agrammon-workbench/agrammon/t/test-data/Models/hr-inclNOxExtendedWithFilters/'                                
> # at /home/christoph/repos/5_GitHub/agrammon-workbench/agrammon/t/run-complex-model-with-variants.t line 46
> # Cannot look up attributes in a Agrammon::Model::Output type object
> ```

@jnthn I'm sorry, I see that it is hard to follow. With the two commits I commented in all `?if Single_extendedOutput` statements except for the one in file `Equides.nhd`. There the statement is including only one output variable:
```
?if Single_extendedOutput

+animals
  print = Equides
  ++units
    en = -
  ++description
    Number of dairy cows in barn.
  ++formula
    Val(animals, Equides::Excretion);

?endif
```
The variable `Val(animals, Equides::Excretion)` does exist and have a valid value...
Anything more that you need to know?

**Comment by jnthn** [comment]
@ChHaeni Oops. Fix in https://github.com/oposs/agrammon/pull/317.

**Comment by jnthn** [comment]
By the way, very much appreciate getting bug reports in the form of a failing test. Thanks! :-)

**Comment by ChHaeni** [comment]
You're welcome :-) Thx for quick fix!
